.github/workflows/dco-check.yml

```diff
@@ -0,0 +1,21 @@
+name: DCO Check
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  dco_check:
+    runs-on: ubuntu-latest
+    name: DCO Check
+    steps:
+    - name: Get PR Commits
+      id: 'get-pr-commits'
+      uses: tim-actions/get-pr-commits@ed97a21c3f83c3417e67a4733ea76887293a2c8f
```

Comment:
Any other projects using this?

Reply:
kata containers and ttrpc-rust using it:

https://github.com/kata-containers/kata-containers/blob/0570317e7b882ab87e72dde9b0539aa41eed4cc0/.github/workflows/commit-message-check.yaml#L29

https://github.com/containerd/ttrpc-rust/blob/e5a5373ca13270b608457944be231ccc71adf756/.github/workflows/sanity.yml#L26